src/quo2/components/list_items/preview_list.cljs

```diff
@@ -0,0 +1,100 @@
+(ns quo2.components.list-items.preview-list
+  (:require [quo.react-native :as rn]
+            [status-im.i18n.i18n :as i18n]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.markdown.text :as quo2.text]))
+
+(def params {32 {:border-radius {:circular 16
+                                 :rounded  10}
+                 :hole-radius   {:circular 18
+                                 :rounded  12}
+                 :margin-left   -8
+                 :hole-size     36
+                 :hole-x        22
+                 :hole-y        -2}
+             24 {:border-radius {:circular 12
+                                 :rounded  8}
+                 :hole-radius   {:circular 13
+                                 :rounded  9}
+                 :margin-left   -4
+                 :hole-size     26
+                 :hole-x        19
+                 :hole-y        -1}
+             16 {:border-radius {:circular 8
+                                 :rounded  8}
+                 :hole-radius   {:circular 9
+                                 :rounded  9}
+                 :margin-left   -4
```

Comment:
Is it ok that `margin-left` has the same value for `16` and `24`?

Reply:
hi @briansztamfater, Thank you very much for reviewing PR.
Yes, both of these items use same left margin
![1](https://user-images.githubusercontent.com/17097240/189546423-ab5b1a70-d43c-417b-89e2-2024e3d7d266.png)
![2](https://user-images.githubusercontent.com/17097240/189546424-ad524016-8f46-4db5-aca8-7e7060ef9ebf.png)